src/test/javascript/portal/visualise/animations/TemporalExtentSpec.js

```diff
@@ -91,6 +91,26 @@ describe("Portal.visualise.animations.TemporalExtent", function() {
         });
     });
 
+    describe('_createDay', function() {
+        var aMomentInTime = moment.utc('2000-01-01T01:00:00.000');
+        var calledWith = null;
+
+        beforeEach(function() {
+            spyOn(temporalExtent, '_setFirstDay').andCallFake(function(date) { calledWith = date; });
+            spyOn(temporalExtent, '_setLastDay').andCallFake(function(date) { calledWith = date; });
+            temporalExtent._createDay(aMomentInTime);
+        });
+
+        it('calls _setFirstDay', function() {
+            expect(temporalExtent._setFirstDay).toHaveBeenCalled();
+            expect(aMomentInTime.toDate()).toEqual(calledWith.toDate());
+        });
+
+        it('calls _setLastDay', function() {
+            expect(temporalExtent._setLastDay).toHaveBeenCalled();
+            expect(aMomentInTime.toDate()).toEqual(calledWith.toDate());
```

Comment:
I'll change that test, yeah. Equality doesn't work for moments for some reason.

Speaking of the package names - yeah, messed as...